<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewFile.txt[*] - Phonoteca</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>459</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuEditing">
    <property name="title">
     <string>Editing</string>
    </property>
    <addaction name="actionProperties"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_new_Artist"/>
    <addaction name="actionAdd_new_Album"/>
    <addaction name="actionAdd_new_Song"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_Item"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuEditing"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionAdd_new_Artist">
   <property name="text">
    <string>Add new Artist</string>
   </property>
  </action>
  <action name="actionAdd_new_Album">
   <property name="text">
    <string>Add new Album</string>
   </property>
  </action>
  <action name="actionAdd_new_Song">
   <property name="text">
    <string>Add new Song</string>
   </property>
  </action>
  <action name="actionDelete_Item">
   <property name="text">
    <string>Delete Item</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
